### PR TITLE
[Agents Maintenance Windows] New Page

### PR DESCRIPTION
**Pull request by presstartgames**

### CODE REVIEW

**Comment by presstartgames** [comment]
@mtrias, Is this going to  🚀  or 💥 ?

**Comment by mtrias** [comment]
Just a minor comment. Make it fly!

File: content/how-to-use-vividcortex/agents-maintence.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+docs: "howto"
+weight: "10"
+---
+
+Agents Maintenance Windows is an enterprise feature which allows organizations to schedule acceptable time windows for
+updating VividCortex agents.
+
+[![Agents Maintenance Windows](/img/docs/agents-maintenance-windows.png)](/img/docs/agents-maintenance-windows.png)
+
+Currently, the supervisor agent periodically checks whether new versions of VividCortex agents are available. This
```
Comment (presstartgames) [inline content/how-to-use-vividcortex/agents-maintence.md]: Updated

**Comment by presstartgames** [comment]
Thanks Miguel!

**Pull request merged**